torch_dicom/testing.py

```diff
@@ -240,10 +261,10 @@ def create_trace_manifest(self, dicom_files: List[Path]) -> pd.DataFrame:
         df = pd.DataFrame(
             {
                 "SOPInstanceUID": sop_uids,
-                "x1": [box.flatten()[0] if box is not None else None for box in trace_coords],
-                "y1": [box.flatten()[1] if box is not None else None for box in trace_coords],
-                "x2": [box.flatten()[2] if box is not None else None for box in trace_coords],
-                "y2": [box.flatten()[3] if box is not None else None for box in trace_coords],
+                "x1": [int(box.flatten()[0].item()) if box is not None else None for box in trace_coords],
```

Comment:
There's a little bit of redundancy here. Maybe something like this would be a bit better?
```python
list_of_coords = [[int(v.item()) for v in box] if box is not None else [None] * 4 for box in trace_coords]
transposed = map(list, zip(*list_of_coords))
coords_dict = {s: l for s, l in zip(["x1", "y1", "x2", "y2"], transposed)}
df = pd.DataFrame(
    {
        ...
    } | coords_dict
```